shared/constants/types/tracker2.tsx

```diff
@@ -88,9 +88,11 @@ export type NonUserDetails = {
 }
 
 export type WebOfTrustEntry = {
-  attestingUser: string
   attestation: string
+  attestingUser: string
+  otherText?: string
```

Comment:
is this ever not defined? might be nice to swap empty strings with `undefined`. or we could change the `other` string to be nullable all the way in avdl. whatevs.